spec/models/news_search_spec.rb

```diff
@@ -5,9 +5,9 @@
 describe NewsSearch do
```

Comment:
The only meaningful change in this file is the following spec:
https://github.com/GSA/search-gov/pull/1125/files#diff-240f9eb9abebe423eeb35fb76e213a2d31cdceae67307a5eb86af5d1c437d540R366

The remaining changes resolve Rubocop offenses.